exo/inference/tinygrad/models/llama.py

```
@@ -48,6 +48,56 @@ def repeat_kv(x: Tensor, n_rep: int) -> Tensor:
+  biases = biases.reshape(*biases.shape, 1).expand(*biases.shape, group_size)
+  return (weight*scales + biases).reshape(weight.shape[0], -1)
+
+
+class AffineQuantizedLinear:
+  def __init__(self, in_features: int, out_features: int, group_size: int = 64, bias: bool = False):
+    self.group_size = group_size
+    self.weight = Tensor.ones(out_features, in_features, dtype=dtypes.uint8)
+    self.scales = Tensor.ones(out_features, in_features // group_size, dtype=dtypes.half)
+    self.biases = Tensor.ones(out_features, in_features // group_size, dtype=dtypes.half)
+
+  def __call__(self, x):
```

Ruff (0.8.2)

Missing return type annotation for special method `__call__`

(ANN204)